etc/profile-m-z/zcat.profile

```diff
@@ -8,6 +8,8 @@ include zcat.local
 #include globals.local
 
 # Allow running kernel config check
+noblacklist ${PATH}/bash
```

Comment:
it looks like that assumes that bash is the shell used with /bin/sh. shouldn't also other shells be allowed? because Debian uses dash as default shell for /bin/sh.

Reply:
I forgot about Debian using dash. To cover all setups it's better to ignore including disable-shell.inc IMO. Thanks for catching this! A fix is waiting for the checks to finish.